langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockCohereEmbeddingModel.java

```diff
@@ -0,0 +1,78 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.bedrock.internal.AbstractBedrockEmbeddingModel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Bedrock Cohere Embedding Model with support for both versions:
+ * {@code cohere.embed-english-v3} and it's multilingual version {@code cohere.embed-multilingual-v3}
+ * <br>
+ * See more details <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-embed.html">here</a>.
+ */
+@SuperBuilder
+@Getter
+public class BedrockCohereEmbeddingModel extends AbstractBedrockEmbeddingModel<BedrockCohereEmbeddingResponse> {
+
+    @Builder.Default
+    private final String model = Types.CohereEmbedEnglishTextV3.getValue();
+
+    @Builder.Default
+    private final String inputType = InputType.SEARCH_DOCUMENT.getValue();
```

Comment:
Is this value mandatory in the request? if not, please remove it, let Cohere set defaults

Reply:
According to official Cohere documentation:
https://cohere.com/blog/introducing-embed-v3
Section: New Mandatory Parameter: Input Type
"The new models have a new required input parameter: input_type, that must be set for every API call and include one of the following four values ..."

Reply:
Ok, let's then make this parameter mandatory, but without a default value, so that user will have to explicitly define it

Reply:
please address this comment

Reply:
done. Removed Lombok @Builder.Default. When someone needs to create/define a model, need to supply InputType as well. Examples provided in `BedrockEmbeddingIT.java`